### PR TITLE
Install rdma-core if infiniband network device is found

### PR DESCRIPTION
**Pull request by rvykydal**
Resolves: rhbz#2050331

### CODE REVIEW

File: pyanaconda/modules/network/network.py [hunk 1]
```diff
@@ -351,6 +351,12 @@ def collect_requirements(self):
                 "teamd",
                 reason="Necessary for network team device configuration."
             ))
+        # infiniband device configuration support
+        if self.get_infiniband_devices():
+            requirements.append(Requirement.for_package(
+                "rdma-core",
```
Comment (poncovka) [inline pyanaconda/modules/network/network.py:357]: Could you add a unit test for this use case, please?

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

**Review by M4rtinK — APPROVED** [review]
Looks good to me as well, thanks! :)

**Pull request merged**